### ISSUE TITLE
When you select a predefined message auto check the input

### ISSUE THREAD

**Issue by ghost** [PrestaShop/PrestaShop#19111]
#### Is your feature request related to a problem?

When you select a predefined message.

The << show customer box >> should be checked

**Comment by colinegin** [comment]
Hello @okom3pom ,

Thanks for your message, but could you give us more details about it ?
Is this in the order page or in the customer service page ?

**Comment by ghost** [comment]
Hello @colinegin 

It's a feature.

When we select a predefined message i think for all merchand is better if the box << show to customer >> is auto checked, because sometimes employees forget to check the box.

![test1](https://user-images.githubusercontent.com/5064590/81909484-f6840100-95ca-11ea-9edb-ae49c48c2ba8.png)
![test2](https://user-images.githubusercontent.com/5064590/81909499-fb48b500-95ca-11ea-8df0-9f4f7a6034a6.png)
![test3](https://user-images.githubusercontent.com/5064590/81909519-01d72c80-95cb-11ea-9c55-0734ba2eb94c.png)

Regards

**Comment by florine2623** [comment]
@okom3pom thanks for the details !
Please stay tuned.

**Comment by florine2623** [comment]
Hi @PrestaShop/prestashop-product-team  ,

The customer would like the message box in an order page to be checked automatically.
Is this a relevant improvement ?

Thanks!

**Comment by MatShir** [comment]
Hi @okom3pom,
It is a nice improvement. It is shared to the team but meanwhile, the project is open source so if anyone you know or anyone who see the issue can do Pull Request (http://doc.prestashop.com/display/PS16/Contributing+code+to+PrestaShop).

**Comment by ghost** [comment]
Hi @MatShir 

I understand your way but it's not compatbile whith law

**Comment by matks** [comment]
> Hi @MatShir
> 
> I understand your way but it's not compatbile whith law

We've discussed this before 😅 in https://github.com/PrestaShop/PrestaShop/issues/19113#issuecomment-629200129

PrestaShop is not here to enforce the law. PrestaShop enables merchants to do a wide range of actions, it's the merchant responsibility to follow the rules that apply to his shop. And these rules change with countries.

We have checked this with a lawyer to be sure 😄
